contracts/config/Config.sol

```diff
@@ -519,6 +533,8 @@ contract Config is ReentrancyGuard, RoleModule, BasicMulticall {
 
         allowedBaseKeys[Keys.IS_GLV_MARKET_DISABLED] = true;
         allowedBaseKeys[Keys.GLV_MAX_MARKET_TOKEN_BALANCE_USD] = true;
+        allowedBaseKeys[Keys.GLV_MAX_CUMULATIVE_DEPOSIT_USD] = true;
```

Comment:
should glvCumulativeDepositUsd be changed to glvEstimatedPoolUsd and glvMaxCumulativeDepositUsd changed to glvMaxDepositUsd?

not sure if i understood what cumulative means correctly